shape_completion_training/src/shape_completion_training/model/filepath_tools.py

```diff
@@ -17,108 +18,99 @@ def unique_trial_name(*names):
     return format_string.format(stamp, sha, *names)
 
 
-def get_trial_directory(base_directory, nick=None, expect_reuse=False, write_summary=True):
+def create_or_load_trial(trial_name=None, params=None, trials_directory=None, write_summary=True):
+    if '/' in trial_name:
```

Comment:
hmm yea I think we cannot infer if the user wants to train or not -- only whether they want to load or not. if there's a '/' in the name we might just be loading because they want to do evaluation. So I say we remove the training return value entirely.

from what I can tell the training return is only used in model runner to set the batch size to 1 vs 16